src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk

```diff
@@ -0,0 +1,310 @@
+
+<h2 class="govuk-heading-l">When not to use</h2>
+
+<p class="govuk-body">Do not use this pattern if you’re just <a href="https://www.gov.uk/guidance/how-to-publish-on-gov-uk/markdown#video"> adding a video to a piece of guidance</a>.</p>
+
+<h2 class="govuk-heading-l">How to design using this pattern</h2>
+
+<div class="govuk-inset-text">Review the whole page each time you make a change to this content so it follows this pattern.</div>
+
+<h3 class="govuk-heading-m">Document type</h3>
+
+<p class="govuk-body">Detailed guide</p>
+
+<h3 class="govuk-heading-m">Title</h3>
+
+<p class="govuk-body">The title should:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+    <li>list the products in the order they appear on the page</li>
+    <li>describe the topic or audience</li>
+</ul>
+
+  {{
+    codeSnippet({
+      code: 'HMRC email updates, videos and webinars [about/for/for the/if] [topic or audience]',
+      canCopy: true
+    })
+  }}
+
+<p class="govuk-body">For example: ‘HMRC email updates, videos and webinars for VAT’.</p>
+
+<h3 class="govuk-heading-m">Summary</h3>
+
+<p class="govuk-body">The summary should start with ‘Learn more about...’ and then explain what the guidance is about.</p>
+
+<p class="govuk-body">For example:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+    <li>‘Learn more about paying tax and keeping records as a property landlord.’</li>
+    <li>‘Learn more about VAT including accounting schemes, VAT returns and keeping records.’</li>
+</ul>
+
+<h3 class="govuk-heading-m">Body</h3>
+
+<p class="govuk-body">Do not include numbers in section headings.</p>
+
+<h4 class="govuk-heading-s">Introductory text</h4>
+
+<p class="govuk-body">Use this introductory text on all pages:</p>
+
+
+  {{
+    codeSnippet({
+      code: '## Subscribe to receive email updates
+
+Subscribe to the [HMRC help and support email service](https://subscriptions.hmrc.gov.uk/) to get information on a wide range of topics for individuals, businesses, employers and agents.
+
+You can also:
+
+* make changes to your topic subscriptions
+* unsubscribe from the service whenever you want
+
+## Registering and joining webinars
+
+You can register for webinars in advance but we recommend you join each webinar 5 minutes before the start.',
+      canCopy: true
+    })
+  }}
+
+<h3 class="govuk-heading-m">How to format the text</h3>
+
+<h4 class="govuk-heading-s">Order of learning products</h4>
+
+<p class="govuk-body">List products in this order:</p>
+
+<ol class="govuk-list govuk-list--number">
+    <li>YouTube videos</li>
+    <li>upcoming webinars</li>
+    <li>recorded webinars</li>
+    <li>e-learning or guidance</li>
+    <li>HMRC community forums</li>
+</ol>
+
+<p class="govuk-body">Use descriptive level 2 headings to tell the user what they will learn.</p>
+
+<p class="govuk-body">Below the heading, describe the action users will take and what they will learn.Then link to the product.</p>
```

Comment:
Should be a space between `learn.` and `Then`

Reply:
Fixed.